### PR TITLE
Android magic window

### PR DESCRIPTION
**Pull request by dmarcos**

### CODE REVIEW

File: src/components/look-controls.js [hunk 1]
```diff
@@ -50,6 +51,24 @@ module.exports.Component = registerComponent('look-controls', {
     if (this.el.sceneEl.is('vr-mode')) { this.onEnterVR(); }
   },
 
+  setupMagicWindowControls: function () {
+    var magicWindowControls;
+    // Only on mobile devices and only enabled if DeviceOrientation permission has been granted.
+    if (utils.device.isMobile()) {
+      magicWindowControls = this.magicWindowControls = new THREE.DeviceOrientationControls(this.magicWindowObject);
+      if (typeof DeviceOrientationEvent === 'undefined' && DeviceOrientationEvent.requestPermission) {
```
Comment (vincentfretin) [inline src/components/look-controls.js]: I think the condition is always false. How can requestPermission exist if DeviceOrientationEvent is undefined?

File: src/components/look-controls.js [hunk 2]
```diff
@@ -50,6 +51,24 @@ module.exports.Component = registerComponent('look-controls', {
...
+        magicWindowControls.enabled = false;
+        if (this.el.sceneEl.components['device-orientation-permission-ui'].premissionGranted) {
```
Comment (vincentfretin) [inline src/components/look-controls.js]: there is a typo here premissionGranted => permissionGranted

File: src/components/look-controls.js (hunk 1 above)
Comment (dmarcos) [reply src/components/look-controls.js]: Thanks!

File: src/components/look-controls.js (hunk 2 above)
Comment (dmarcos) [reply src/components/look-controls.js]: Good catch!

**Pull request merged**